lib/school_house_web/templates/layout/_lesson_menu.html.leex

```diff
@@ -2,287 +2,343 @@
+                    </li>
+                    <li>
+                        <%= lesson_link @conn, :basics, :enum do %>
+                            <%= gettext("Enum") %>
+                        <% end %>
+                    </li>
+                    <li>
+                        <%= lesson_link @conn, :basics, :pattern_matching do %>
+                            <%= gettext("Pattern Matching") %>
+                        <% end %>
+                    </li>
+                    <li>
+                        <%= lesson_link @conn, :basics, :control_structures do %>
+                            <%= gettext("Control Structures") %>
+                        <% end %>
+                    </li>
+                    <li>
+                        <%= lesson_link @conn, :basics, :functions do %>
+                            <%= gettext("Functions") %>
+                        <% end %>
+                    </li>
+                    <li>
+                        <%= lesson_link @conn, :basics, :pipe_operator do %>
+                            <%= gettext("Pipe Operator") %>
+                        <% end %>
+                    </li>
+                    <li>
+                        <%= lesson_link @conn, :basics, :modules do %>
+                            <%= gettext("Modules") %>
+                        <% end %>
+                    </li>
+                    <li>
+                        <%= lesson_link @conn, :basics, :mix do %>
+                            <%= gettext("Mix") %>
+                        <% end %>
+                    </li>
+                    <li>
+                        <%= lesson_link @conn, :basics, :sigils do %>
+                            <%= gettext("Sigils") %>
+                        <% end %>
+                    </li>
+                    <li>
+                        <%= lesson_link @conn, :basics, :documentation do %>
+                            <%= gettext("Documentation") %>
+                        <% end %>
+                    </li>
+                    <li>
+                        <%= lesson_link @conn, :basics, :comprehensions do %>
+                            <%= gettext("Comprehensions") %>
+                        <% end %>
+                    </li>
+                    <li>
+                        <%= lesson_link @conn, :basics, :strings do %>
+                            <%= gettext("Strings") %>
+                        <% end %>
+                    </li>
+                    <li>
+                        <%= lesson_link @conn, :basics, :date_time do %>
+                            <%= gettext("Date and Time") %>
+                        <% end %>
+                    </li>
+                    <li>
+                        <%= lesson_link @conn, :basics, :iex_helpers do %>
+                            <%= gettext("IEX Helpers") %>
+                        <% end %>
+                    </li>
+                </ol>
+            </li>
+
+        </ul>
+
+        <!-- Second column -->
```

Comment:
This is really smart. I need to get in the habit of commenting my markup to make it easier to grok later.

Reply:
Yes, it took me a while to understand the structure, I hope that makes it easier.

Reply:
I will make a effect to incorporate this moving forward to ease contribution friction. Thanks for the tip @jaimeiniesta